### PR TITLE
Updating Samples

### PR DESCRIPTION
**Pull request by john0005**
@tschie Can you look this over?
issues: 
  - https://devtopia.esri.com/runtime/java-se-sdk/issues/1055 or https://github.com/Esri/arcgis-runtime-samples-java/issues/113
 - https://github.com/Esri/arcgis-runtime-samples-java/issues/117

### CODE REVIEW

File: src/main/java/com/esri/samples/editing/update_geometries/UpdateGeometriesSample.java [hunk 1]
```diff
@@ -108,8 +109,11 @@ public void start(Stage stage) throws Exception {
                 featureLayer.clearSelection();
                 featureLayer.selectFeature(selected);
               } else {
-                // move selected features
-                moveSelected(mapPoint);
+                FeatureQueryResult result = featureLayer.getSelectedFeaturesAsync().get();
```
Comment (tschie) [inline src/main/java/com/esri/samples/editing/update_geometries/UpdateGeometriesSample.java]: I don't think we should use a blocking `get` here. Also, we shouldn't keep track of a `selected` feature if we're just going to check for it every time.

File: src/main/java/com/esri/samples/symbology/symbol_dictionary/SymbolDictionaryController.java [hunk 2]
```diff
@@ -74,14 +79,21 @@ private void handleSearchAction() {
...
-        List<SymbolStyleSearchResult> symbolResults = searchResult.get();
+        symbolResults = searchResult.get();
 
         Platform.runLater(() -> {
-          // create and add results to listview
-          symbolResults.stream().map(SymbolView::new).collect(Collectors.toCollection(() -> resultList.getItems()));
+          int results = symbolResults.size();
+          if (results > 0) {
+            // create and add results to listview
+            symbolResults.subList(0, Math.min(results, RESULT_SIZE)).stream().map(SymbolView::new).collect(Collectors.toCollection(() -> resultList.getItems()));
```
Comment (tschie) [inline src/main/java/com/esri/samples/symbology/symbol_dictionary/SymbolDictionaryController.java]: There's an official `Pagination` control for this that you should use: http://docs.oracle.com/javase/8/javafx/api/javafx/scene/control/Pagination.html.

**Review by tschie — CHANGES_REQUESTED** [review]
Added some suggestions.

File: src/main/java/com/esri/samples/editing/update_geometries/UpdateGeometriesSample.java (hunk 1 above)
Comment (tschie) [reply src/main/java/com/esri/samples/editing/update_geometries/UpdateGeometriesSample.java]: @john0005 Can you remove changes for this sample? I have a fix that handles a couple other things as well in another PR: https://github.com/Esri/arcgis-runtime-samples-java/pull/176.

**Comment by john0005** [comment]
@tschie Can you look this over again added the Pagination Class.

**Review by tschie — APPROVED** [review]

**Pull request merged**